### ISSUE TITLE
MOPs Spring Modifier error on node spawn

### ISSUE THREAD

**Issue by tedatsavages** [toadstorm/MOPS#239]
Houdini consistently returns the following error on 348 when creating a MOPs Spring Modifier. Something cooks initially if the playhead is off the first frame, but it won't cook again, and nothing is visible in the viewport. MOPs v1.06.

Error:
Traceback (most recent call last):
  File "opdef:/MOPS::Sop/Spring_Modifier::1.0?OnCreated", line 13, in <module>
AttributeError: 'module' object has no attribute 'viewport_selection'

**Comment by tedatsavages** [comment]
Just fixed it by flagging the output - it was flagging the solver with the blasts as the node output.

**Comment by toadstorm** [comment]
I can't recreate this. The module not being found makes me suspect an installation problem, maybe? Changing the display node inside the HDA shouldn't do anything when an Output SOP is involved... this overrides any flags.

**Comment by FinalFeliz** [comment]
I just ran into this issue using Houdini 18.0.416 and MOPs 1.0.6.
Diving into the HDA I found there was a missing "s" on line 1 in:
MOPs_Spring_Modifier/spring_solver/DOP Network.
In the Timestep attribute, the expression references ch("substep", but should be "substeps"
Line should be:
ch("timescale")/(ch("substeps")*$FPS)
Editing this line gets rid of the error, but trying to get it to work leads to very slow performance and there is no animation, I have an animated falloff node preceding this, it was working, but not after putting the spring node in.
